src/app/functions/server/changeSubscription.ts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Subscriber } from "../../(nextjs_migration)/(authenticated)/user/breaches/breaches";
+import { updateFxAProfileData } from "../../../db/tables/subscribers";
+
+const MONITOR_PREMIUM_CAPABILITY = "monitor";
+
+export async function changeSubscription(
+  subscriber: Subscriber,
+  enabled: boolean,
+) {
+  const currentFxAProfile = subscriber?.fxa_profile_json as FxaProfile;
+  let subscriptions = currentFxAProfile.subscriptions ?? [];
```

Comment:
if `subscribers` can be nullish, would `currentFxAProfile` be null/undefined and therefore `currentFxAProfile.subscriptions` could fail if we access `.subscriptions` on an undefined object?